README.md

```diff
@@ -199,3 +182,31 @@ If you are using Avro format for producing data to Kafka, here is the correspond
 ```
 
 If you are not using Avro format for producing data to Kafka, there will be no schema in Confluent Schema Registry.
+
+# Publishing Docker Images
+
+*Note: The following instructions are only relevant if you are an administrator of this repository and have push access to the https://hub.docker.com/r/cnfldemos/kafka-connect-datagen/ registry.  The local Docker daemon must be logged into a proper Docker Hub account.*
```

Comment:
registry --> repository

Reply:
In my experience Registry is preferrend nomenclature for Docker images: https://docs.docker.com/registry/

Reply:
Maybe that's preferrable for the service itself, but repository is better for the collection of images.  I'm indifferent and will change it up